ftw/recipe/deployment/README.txt

```diff
@@ -835,6 +837,66 @@ Our packall script should contain pack commands for all storages::
     /sample-buildout/bin/zeopack -S storage1 \
         && echo `date +%Y-%m-%dT%H:%M:%S%z` "packed storage1" >> /sample-buildout/var/log/pack.log
 
+
+Create a buildout with a two filestorage parts, two zeo parts and custom zeo-script-name::
+
+    >>> write('buildout.cfg',
+    ... """
+    ... [buildout]
+    ... develop = plone.recipe.zope2instance plone.recipe.zeoserver collective.recipe.filestorage
+    ... parts = instance1 zeo filestorage zeo2 filestorage2 deployment
+    ...
+    ... [instance1]
+    ... recipe = plone.recipe.zope2instance
+    ...
+    ... [zeo]
+    ... recipe = plone.recipe.zeoserver
+    ...
+    ... [filestorage]
+    ... recipe = collective.recipe.filestorage
+    ... parts = storage1
+    ...
+    ... [zeo2]
+    ... recipe = plone.recipe.zeoserver
+    ... zeopack-script-name = zeopack2
```

Comment:
Hmm. If you forget to set `zeopack-script-name` for the second storage, it will default to `zeopack` and therefore overwrite the script for the first storage (but still invoke it twice from the `packall` script). Not really a fan of that, that would cause quite a mess :-/

What about instead completely auto-generating the zeopack script name, without letting the user control it? For example by setting it to `zeopack-{zeo_part_name}` (at least for zeos 2-n).

Another possibility would be to keep the behavior of defaulting to `zeopack`, but ensuring that each zeo part has a unique `zeopack-script-name`, and refusing to buildout otherwise.